build_sysext

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+# Copyright (c) 2023 Flatcar Linux. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file. Script to generate sysext. See systemd-sysext(8).
+# Prerequisite that you've run build_packages and build_image
+
+SCRIPT_ROOT=$(dirname "$(readlink -f "$0")")
+. "${SCRIPT_ROOT}/common.sh" || exit 1
+
+# Script must run inside the chroot
+assert_inside_chroot
+assert_root_user
+
+DEFINE_string board "${DEFAULT_BOARD}" \
+  "The board to build a sysext for."
+
+FLAGS_HELP="USAGE: build_sysext [flags] [sysext name] [packages to build].
+This script is used to build a Flatcar sysext. 
+
+Examples:
+
+Build a systemd-sysext image named command-extras with package_1, package_2 and package_3 for amd64:
+sudo build_sysext --board=amd64-usr command-extras package_1 package_2 package_3
+
+Builds a systemd-sysext image named command-extras with package_1, package_2 and package_3 for arm64:
+sudo build_sysext --board=arm64-usr command-extras package_1 package_2 package_3
+
+"
+
+show_help_if_requested "$@"
+
+# Parse command line
+FLAGS "$@" || exit 1
+
+eval set -- "${FLAGS_ARGV:-prod}"
+
+. "${BUILD_LIBRARY_DIR}/release_util.sh" || exit 1
+
+. "${BUILD_LIBRARY_DIR}/toolchain_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/board_options.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/build_image_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/prod_image_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/dev_container_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/test_image_content.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/torcx_manifest.sh" || exit 1
```

Comment:
Why is this imported? Please only import what you use.
Also note that if you want to rely on a flag that is defined here, the import would need to be moved before the flag parsing. For example, if the user should be able to set `BUILD_DIR`.

Reply:
What we need for sure is first `build_library/board_options.sh` and then `build_library/build_image_util.sh` to set up `BUILD_DIR`. And `build_library/oem_sysext_util.sh` for the helper.
What about the rest, where is it used?

Reply:
Please try to reduce this list to only those that get used